criu/net.c

```diff
@@ -111,15 +111,18 @@ int read_ns_sys_file(char *path, char *buf, int len)
 	}
 
 	rlen = read(fd, buf, len);
+	if (rlen == -1)
+		pr_perror("Can't read ns' %s", path);
 	close(fd);
 
 	if (rlen == len) {
+		buf[len - 1] = 0;
 		pr_err("Too small buffer to read ns sys file %s\n", path);
 		return -1;
```

Comment:
Does it make sense to return a partial buffer? I mean, maybe it's better to do `buf[0] = 0`.